shub/config.py

```diff
@@ -28,6 +28,8 @@ def __init__(self):
         }
         self.apikeys = {}
         self.version = 'AUTO'
+        self.stacks = {}
+        self.requirements_file = None
```

Comment:
let's agree to default to None. if users would like to use `requirements.txt` - that's 1 line in config. `requirements-scrapinghub.txt` is ugly.